lib/content-services/tree-view/components/tree-view.component.ts

```diff
@@ -38,6 +38,9 @@ export class TreeViewComponent implements OnInit, OnChanges {
     @Output()
     nodeClicked: EventEmitter<NodeEntry> = new EventEmitter();
 
+    @Output()
```

Comment:
can you add the description of this output here. The doc generator get it from here .